reconcile/queries.py

```diff
@@ -106,6 +106,7 @@ def get_secret_reader_settings() -> Optional[Mapping[str, Any]]:
       readTimeout
       connectTimeout
     }
+    cloudflareEmailDomainAllowList
```

Comment:
We recently adjusted the dev-guidelines to unify our approach for gql queries: https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#generated-graphql-data-classes

I think in this case you do not need the `queries` module any longer. You could add a new query for settings, like done for the smtp use-case: https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/gql_definitions/common/smtp_client_settings.gql

You could also add the vault-settings there, as you will require them for the secret reader too.